packages/test/test-service-load/src/loadTestDataStore.ts

```diff
@@ -573,27 +573,20 @@ class LoadTestDataStore extends DataObject implements ILoadTest {
 			config.testConfig.useVariableOpSize === true
 				? Math.floor(Math.random() * opSizeinBytes)
 				: opSizeinBytes;
+		const largeOpRate = config.testConfig.largeOpRate ?? 1;
+		let opsSent = 0;
 
-		const sendSingleOp =
-			opSizeinBytes === 0
-				? () => {
-						dataModel.counter.increment(1);
-				  }
-				: () => {
-						const opPayload = generateContentOfSize(getOpSizeInBytes());
-						const opKey = Math.random().toString();
-						dataModel.sharedmap.set(opKey, opPayload);
-				  };
+		const sendSingleOp = () => {
+			if (opSizeinBytes > 0 && largeOpRate > 0 && opsSent % largeOpRate === 1) {
+				const opPayload = generateContentOfSize(getOpSizeInBytes());
+				const opKey = Math.random().toString();
+				dataModel.sharedmap.set(opKey, opPayload);
+			} else {
+				dataModel.counter.increment(1);
```

Comment:
I think you want to do both. I.e. if you take first branch always, will test complete? Maybe it does today (I remember completion logic was weird), but I think we should expect some number of increments on the counter as a main thing here, and thus we should keep counting...

Reply:
The test completes, see https://dev.azure.com/fluidframework/internal/_build/results?buildId=127829&view=results

I believe the completion logic relies on the opsSent variable: https://github.com/microsoft/FluidFramework/blob/main/packages/test/test-service-load/src/loadTestDataStore.ts#L629

This needs to be decoupled, as if we only rely on the counter value (setting aside the problem of using the subject of the test for coordination), the large ops will not be accounted for, so the test would technically send 20 extra ops per each client.